project/env_hls.tcl

```diff
@@ -13,3 +13,12 @@ if { ([string first "vitis" $exe] > -1) && ($year > 2019) } {
 	interp alias {} cosim_design {} interp invokehidden {} -- cosim_design -verbose
 }
 
+
+if {[regexp Vivado $exepath tool]} {
+  set VitisOrVivado "_Vivado"
```

Comment:
I'd suggest setting VitisOrVivado to an empty string, for case where Vivado HLS is being used. This will minimise the risk of breaking anything with the current HLS code and FW chains.